### ISSUE TITLE
Accessibility issues: Automatically determine WCAG criterionas and acceptance levels

### ISSUE THREAD

**Issue by tskarhed** [grafana/auto-triager#5]
We use [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/)to evaluate the accessibility for Grafana. Quite often we can determine that an issue is accessibility related, and from a fix point of view it is cumbersome to find which [WCAG criterion](https://www.w3.org/TR/WCAG21/) it applies to, hence it is rarely done.

Furthermore, some customers ask about conformance levels (A, AA, AAA). We have no way of automatically pulling the data and say for sure how much we are conforming. It all has to be done manually.

By automating it, we can have a live view of our conformance level.

Here is an example of correct labels on an issue
https://github.com/grafana/grafana/issues/82172

**Comment by academo** [comment]
@tskarhed can you please expand on the process to determine which criterion fits? This is essential to be able to automate it. 

If you can provide examples steps by step for worse, normal, good and best case scenarios we can have an idea of the kind of work required to automate it.

**Comment by tskarhed** [comment]
Usually by going through the hierarchy in the table of contents https://www.w3.org/TR/WCAG21/

For example for anything related to content order or hierarchy would probably go under a criterion under 1.3.

Things that are hard to see or differentiate between two visual states would go under 1.4.

Issues or bugs with interactive elements go under section 2, with keyboard accessible being 2.1.



Each success criterion has a conformance level, and this one we should basically just take. For example 2.1.4 has Level A.

So to summarize:

1. Read through the issue and identify theme (keyboard, contrast, interaction etc.)
2. Locate the rough category on the WCAG criterion list
3. Read through each criterion to see which is more suiting
